docs/lib-documentation.md

```diff
@@ -0,0 +1,55 @@
+Kurtosis Lambda API Lib Documentation
+=====================================
+This documentation describes all the necessary objects to configure, create and run a custom implementation of a Kurtosis Lambda.
+
+
+Kurtosis Lambda
+---------------
+This interface defines the behavior of a Kurtosis Lambda which is a way to package and make available preconfigured services that users can easily load and use in their Kurtosis tests.
+
+### Execute([NetworkContext][networkcontext] networkContext, String serializedParams) -\> (String serializedResult)
+This function executes the only functionality that contains an implementation of a Kurtosis Lambda.
```

Comment:
better is, "executes the functionality contained inside this implementation of Kurtosis Lambda."